utils/scale-test

```diff
@@ -38,20 +38,23 @@ def has_debuginfo(swiftc):
 
 
 def write_input_file(args, ast, d, n):
-    ifile = os.path.join(d, "in%d.swift" % n)
-    with open(ifile,'w+') as f:
+    fname = "in%d.swift" % n
+    pathname = os.path.join(d, fname)
+    with open(pathname,'w+') as f:
```

Comment:
Interesting. I see the results in the jenkins job, but when I install and run flake8 locally I get no errors. Is there some particular version or configuration of flake8 to install to get such results?